requirements.txt

```diff
@@ -1,2 +1,3 @@
 PyYAML==3.11
+yapf==0.10.0
```

Comment:
I considered this, the problem is that most people who install this code will want to run the tests to validate any artifacts they write. Which will now break if this isn't installed. The way we solve this in grr is to have a separate package with test dependencies, but that is overkill in this case. I think adding a small dependency here is less bad than having tests break unexpectedly. I don't feel too strongly though, I can install it separately in the travis config if you like.